srcpkgs/tlp/files/README.voidlinux

```diff
@@ -0,0 +1,7 @@
+### TLP Profiles Daemon ###
+
+You can either use the shipped runit service, see also:
+
+ "https://docs.voidlinux.org/config/services/index.html",
+
+or manually launch 'tlp-pd-pkexec' on startup of your DE.
```

Comment:
This file is superfluous if the `tlp-pd-pkexec` script is removed, as recommended below. Users should expect that they can run the service that comes with a package, or they can read the upstream documentation to understand that they should launch `tlp-pd`.